src/platform_impl/windows/event_loop.rs

```diff
@@ -583,6 +620,102 @@ fn main_thread_id() -> u32 {
     unsafe { MAIN_THREAD_ID }
 }
 
+/// A screenshot from the toolhelp tool.
+struct ToolhelpSnapshot {
+    /// The handle to the snapshot.
+    handle: OwnedHandle,
+
+    /// Are we returning the first thread entry?
+    first_entry: bool,
+}
+
+impl ToolhelpSnapshot {
+    /// Take a screenshot of the provided process.
+    fn new(process_id: u32) -> Result<Self, EventLoopError> {
+        // Take a snapshot.
+        let handle =
+            unsafe { toolhelp::CreateToolhelp32Snapshot(toolhelp::TH32CS_SNAPTHREAD, process_id) };
```

Comment:
You can pass `0` to `CreateToolhelp32Snapshot`, no need to pass the current process ID.

Reply:
We use the process ID in other places, so I figure why not.

Reply:
According to the docs:

> This parameter can be zero to indicate the current process.

So it's not really necessary.

Reply:
I figure it's slightly clearer to whoever is reading who may not be familiar with this API.